<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>OMRON_demoClass</class>
 <widget class="QMainWindow" name="OMRON_demoClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>984</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>OMRON_demo</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <widget class="QPushButton" name="ConnectBtn">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>50</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>连接</string>
    </property>
   </widget>
   <widget class="QLabel" name="displayLabel">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>0</y>
      <width>650</width>
      <height>410</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Panel</enum>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QPushButton" name="DisconnectBtn">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>50</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>断开连接</string>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="whiteBalanceDoubleSpinBox">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>200</y>
      <width>62</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="exposureDoubleSpinBox">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>280</y>
      <width>81</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>200</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>白平衡</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>240</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>增益</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_3">
    <property name="geometry">
     <rect>
      <x>760</x>
      <y>280</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>曝光</string>
    </property>
   </widget>
   <widget class="QSpinBox" name="gainSpinBox">
    <property name="geometry">
     <rect>
      <x>830</x>
      <y>240</y>
      <width>42</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QCheckBox" name="AWB_btn">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>200</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>自动</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="AGC_btn">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>240</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>自动</string>
    </property>
   </widget>
   <widget class="QCheckBox" name="AE_btn">
    <property name="geometry">
     <rect>
      <x>670</x>
      <y>280</y>
      <width>81</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>自动</string>
    </property>
   </widget>
   <widget class="QPushButton" name="pathbtn">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>90</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>文件选择</string>
    </property>
   </widget>
   <widget class="QPushButton" name="captureSinglebtn">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>90</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>单次拍照</string>
    </property>
   </widget>
   <widget class="QPushButton" name="startcapturebtn">
    <property name="geometry">
     <rect>
      <x>690</x>
      <y>130</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>连续拍照</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_4">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>160</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>拍照间隔</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="timelineEdit">
    <property name="geometry">
     <rect>
      <x>770</x>
      <y>160</y>
      <width>41</width>
      <height>23</height>
     </rect>
    </property>
   </widget>
   <widget class="QComboBox" name="unitcomboBox">
    <property name="geometry">
     <rect>
      <x>820</x>
      <y>160</y>
      <width>72</width>
      <height>24</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="stopcapturebtn">
    <property name="geometry">
     <rect>
      <x>800</x>
      <y>130</y>
      <width>80</width>
      <height>24</height>
     </rect>
    </property>
    <property name="text">
     <string>停止拍照</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>900</x>
      <y>160</y>
      <width>71</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>时间单位</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>984</width>
     <height>21</height>
    </rect>
   </property>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="OMRON_demo.qrc"/>
 </resources>
 <connections/>
</ui>
